struct.go

```diff
@@ -0,0 +1,73 @@
+package gofakeit
+
+import (
+	"reflect"
+)
+
+// Struct fills in elements of a struct with random data
+// based on the value of `fake` tag of exported elements
+func Struct(v interface{}) {
+	r(reflect.TypeOf(v), reflect.ValueOf(v), "")
+}
+
+func r(t reflect.Type, v reflect.Value, template string) {
+	switch t.Kind() {
+	case reflect.Ptr:
```

Comment:
Code climate believes this function is too "complex". Although its clean to me. Let make it happy by separating it into a private functions for ones that are more than just one liners.

Reply:
@brianvoe This is one of those general rules which needs to be broken.
IMHO, doing this will make the code much more 'complex' and unnecessarily verbose and difficult to read. 
Tools like Code Climate apply a generic set of rules without any context or intelligence. I feel they are more of an assistant who can help avoid simple and silly mistakes rather than strict set of rules to follow.  Sorry for my rant. 😐 

I can update it if that's a strict criteria for pushing code into master.

Reply:
Ok we can keep it as is for now.